src/bin/compress.rs

```diff
@@ -36,8 +34,8 @@ pub struct Args {
     pub truncate: Option<usize>,
 
     /// use dreamcoder format
```

Comment:
could  you rewrite the doc comment (and list the options in it / maybe point to an example file of each type for total clarity?)

Reply:
Fixed in [docs: updated docstring for fmt flag](https://github.com/mlb2251/stitch/pull/68/commits/2b7dfd51c6036b8416014e5f9f89b1e96760848c).